services/oc-pmrmrg/service.cli/execute.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+# set sh strict mode
+set -o errexit
+set -o nounset
+IFS=$(printf '\n\t')
+
+cd /home/scu/pmr_mrg
```

Comment:
could be replaced by (since this is executed as user scu already)
cd ~/pmr_mrg

Reply:
yep